libc/test/src/math/smoke/CMakeLists.txt

```
@@ -3562,6 +3562,19 @@ add_fp_unittest(
     libc.src.__support.macros.properties.types
 )
 
+add_fp_unittest(
+  fmaf16_test
+  SUITE
+    libc-math-smoke-tests
+  SRCS
+    fmaf16_test.cpp
+  HDRS
+    FmaTest.h
+  DEPENDS
+    libc.src.math.fmaf16
+    libc.src.__support.macros.properties.types
+)
```

```suggestion
  DEPENDS
    libc.src.math.fmaf16
    libc.src.__support.CPP.type_traits
    libc.src.__support.FPUtil.cast
    libc.src.__support.macros.properties.types
)
```

Or maybe we should fix this now that we have `fputil::cast`: https://github.com/llvm/llvm-project/blob/3e6f618e86f5fbad2c2d5802416ec3d3366a2837/libc/test/src/math/smoke/FmaTest.h#L93-L98

Then FmaTest.h would only need `libc.src.__support.FPUtil.cast`.

---

Hi @overmighty , how can I fix it? Sorry I am not understand it. :-)

---

Hi @overmighty , I have updated it, please take a look. :-)